services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -778,6 +775,27 @@ public void installDropFlow(final DatapathId dpid) throws SwitchOperationExcepti
         }
     }
 
+    private boolean isSwitchSupportsMeters(IOFSwitch sw) {
+        if (OVS_MANUFACTURER.equals(sw.getSwitchDescription().getManufacturerDescription())) {
+            logger.info("Meters are not supported on OVS switch {}", sw.getId());
```

Comment:
Producing log message here will increate mess in our logs. It must be exception... and caller must decide is it going to log it... to ignore it... or handle in some other way.

Imagine several call to isSwitchSupportsMeters during one operation/command.